virt/node_maintenance/virt-managing-node-labeling-obsolete-cpu-models.adoc

```diff
@@ -6,7 +6,8 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-You can schedule a virtual machine (VM) on a node where the CPU model and policy attribute of the VM are compatible with the CPU models and policy attributes that the node supports. By specifying a list of obsolete CPU models in a xref:../../authentication/configmaps.adoc#authentication-configmap-overview_builds-configmaps[config map], you can exclude them from the list of labels created for CPU models.
+You can schedule a virtual machine (VM) on a node where the CPU model and policy attribute of the VM are compatible with the CPU models and policy attributes that the node supports.
```

Comment:
Suggestion: You can schedule a virtual machine (VM) on a node as long as the VM CPU model and policy are supported by the node.

I think that makes the requirement a little clearer.

Reply:
I absolutely agree, this is a great edit. Thanks!